### PR TITLE
chore(): update version angular 13 and other improvements

### PR DESCRIPTION
**Pull request by Tony133**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Docs
- [x] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
- Update version angular 13 ❤️ and other improvements

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
This PR, update version angular 13 ❤️ and other improvements.

### CODE REVIEW

**Comment by kamilmysliwiec** [comment]
> Migration to eslint (if necessary, however, I remove the commit with the migration to eslint and use the one I did some time ago)

Yeah, let's stick to the `ng lint`. They (Angular team) will migrate, eventually.

**Comment by Tony133** [comment]
Ok, I have removed the commit for the migration to `eslint` and I have updated the description of the PR.

File: package.json [hunk 1]
```diff
@@ -20,32 +20,32 @@
...
+    "@angular/router": "13.1.1",
+    "core-js": "3.8.0",
     "hammerjs": "2.0.8",
     "ngx-perfect-scrollbar": "10.1.0",
     "nodemon": "2.0.7",
     "prismjs": "1.25.0",
-    "rxjs": "6.6.6",
-    "rxjs-compat": "6.6.6",
-    "tslib": "2.1.0",
+    "rxjs": "^7.4.0",
```
Comment (kamilmysliwiec) [inline package.json:39]:
```suggestion
    "rxjs": "7.4.0",
```

**Comment by Tony133** [comment]
I forgot, it gives some warnings from webpack-dev-server about compiling some sass files but I can't figure out how to fix them:

screenshot

**Comment by kamilmysliwiec** [comment]
LGTM

**Pull request merged**